Gemfile

```diff
@@ -92,6 +92,9 @@ gem 'fhir_models'
 # Split-arch queue support
 gem 'redis-queue'
 
+# JSON Patch and JSON Pointer implementation
+gem 'hana'
```

Comment:
Whoa I follow the guy who made this on Twitter!

On a separate note, I'm guessing you noticed that FHIR allows JSON Patch, but allows [FHIRPath Patch](https://www.hl7.org/fhir/fhirpatch.html). I think FHIRPath would be cool to use for this, but also I'm virtually certain their is no existing Ruby implementation, and `hana` has 4 million downloads. So I agree with how you've implemented this and am not requesting any changes here, but just wanted to check in with you on your thought process towards using this approach.

Reply:
Yeah, I went with the JSON Patch for basically the reason you said, it just seemed like it was more broadly supported/used. I also felt that JSON Patch would be more approachable for someone who is unfamiliar to FHIR, while being easy enough to understand for someone deeply familiar with FHIR.

Reply:
That being said, I also think it might be cool to support this using FHIR alone as well, and there's no reason why we can't add that on later if we choose.

Reply:
Cool, I'm on board